.github/actions/ghcr-prune-3.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python3
+# PYTHON_ARGCOMPLETE_OK
+import argparse
+import dateutil.parser
+import getpass
+import os
+import requests
+from datetime import datetime, timedelta
+
+__author__ = "Max VeRBiTSKiy"
+__version__ = "0.1"
+__copyright__ = "Copyright (C) 2021 Fiona Klute"
+__license__ = "MIT"
+
+# GitHub API documentation: https://docs.github.com/en/rest/reference/packages
+github_api_accept = 'application/vnd.github.v3+json'
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description='List versions of a GHCR container image you own, and '
+        'optionally delete (prune) old, untagged versions.')
+    #parser.add_argument('--token', '-t', action='store_true',
+                        #help='ask for token input instead of using the '
+                        #'GHCR_TOKEN environment variable')
+    parser.add_argument('--container', default='crash-js-app',
+                        help='name of the container image')
+    parser.add_argument('--verbose', '-v', action='store_true',
+                        help='print extra debug info')
+    #parser.add_argument('--prune-age', type=float, metavar='DAYS',
+                        #default=None,
+                        #help='delete untagged images older than DAYS days')
+    parser.add_argument('--dry-run', '-n', action='store_true',
+                        help='do not actually prune images, just list which '
+                        'would be pruned')
+    parser.add_argument('--number', type=int, metavar='COUNT', 
+                        default=None,
+                        help='delete all images instead of COUNT last')
+    parser.add_argument('--token', '-t',
+                        help='add a github token to connect')
+
+    # enable bash completion if argcomplete is available
+    try:
+        import argcomplete
+        argcomplete.autocomplete(parser)
+    except ImportError:
+        pass
+
+    args = parser.parse_args()
+
+    token = args.token
+    #if args.token:
+        #token = getpass.getpass('Enter Token: ')
+    #elif 'GHCR_TOKEN' in os.environ:
+        #token = os.environ['GITHUB_TOKEN']
+    #else:
```

Comment:
it's better to keep GITHUB_TOKEN as environment variable
it would be easier to use it in actions